.github/workflows/integration-tests.yml

```diff
@@ -36,7 +36,8 @@ jobs:
       - name: Start docker compose in the background
         run: docker compose up -d
       - name: Test that client can ping resource
-        run: docker compose exec -it client ping 172.20.0.100 -c 10
+        run: docker compose exec -it client timeout 60 bash -c 'until ping -W 1 -c 1 172.20.0.100 &>/dev/null; do true; done'
```

Comment:
@conectado I went ahead and applied the `ping until success or timeout` logic here as we chatted about :-).